README.md

```diff
@@ -52,8 +52,12 @@ def build():
   <summary><code>envdlib.tensorboard</code> is defined in <a href="https://github.com/tensorchord/envdlib/blob/main/src/monitoring.envd">github.com/tensorchord/envdlib</a></summary>
 
 ```python
-def tensorboard(envd_port=6006, envd_dir="/home/envd/logs",
-        host_port=0, host_dir="/var/log/tensorboard"):
+def tensorboard(
+    envd_port=6006,
+    envd_dir="/home/envd/logs",
+    host_port=0,
+    host_dir="/tmp",
```

Comment:
Check https://github.com/tensorchord/envdlib/pull/11

I'm not sure which dir should we use.

Reply:
Shall we provide function `empty_dir()` like k8s?

Reply:
SGTM. Should it create a folder under `.cache/envd`? We can open an issue to track this. I guess `/tmp` works for now. WDYT

cc @gaocegege

Reply:
We can open an issue to keep track. /tmp LGTM for now.

Reply:
So can we merge this one?